### ISSUE TITLE
Addon no longer works on Chrome.

### ISSUE THREAD

**Issue by xerogabe106** [xternal7/ultrawidify#9]
Its just not working anymore. I'm happy to give any info you need to help make it work again. Also happy to test anything. 

Thanks!

**Comment by xternal7** [comment]
Did it only stop working on Netflix ([known issue](https://github.com/xternal7/ultrawidify/issues/8) or youtube as well?

Because if it's youtube, I can't reproduce. 

Can I have Chrome version, your OS and your other extensions?

**Comment by xerogabe106** [comment]
It doesn't work for either. I mainly wanted it for Netflix, so I didn't really poke around youtube very much. But it certainly did not work on Netflix. 

Chrome Version 61.0.3163.100

OS: Win10. 

Extensions: uBlock Origin 1.14.12

**Comment by xternal7** [comment]
I'm sorry for taking my sweet time to respond, but if your offer to test stuff still stands:

[Here's a Chrome build of whatever's on Github right now](http://tamius.net/ultrawidify/versions-git/ultrawidify-git-2.0a1.crx) — so, development version. Not quite ready for the mainstream.

**TL;DR:** ui is gone, toolbar button is useless, you have to use your keyboard to change aspect ratio. Autodetection doesn't work in Chrome.

Changes:
* UI buttons are gone
* Keyboard shortcuts remain the same — unless you changed them, in which case here's the default shortcuts:

```
w - fit to width
h - fit to height
r - reset
s, d, x, c - should make video fit the screen perfectly, if the source video is in the following aspect ratios: 16:9, 21:9, 16:10, 4:3. Not sure about the order.
```
Zoom, unzoom binds are gone.

`a` is still reserved for autodetection, more on that later.


* It _should_ work on youtube and vimeo. It should also work on netflix (in theory at least, as long as the video is in full screen mode), but **I can't test that yet.** Vid.me is a mixed bag. I think it should work on embedded videos? Sometimes it just crashes vid.me though.

* In general, it only works in full screen mode (exception: vimeo). 

* Autodetection _doesn't_ work in Chrome for some reason. It's not utterly broken — autodetection _does_ run in chrome, except that the code gives utterly bogus results. (No such problem in Firefox, though?)

* That extension button in the tab bar has no functionality yet. If it looks ugly and broken and if nothing works — that's because I've only begun working on it.

**Comment by xternal7** [comment]
With release of 2.0.2, Chrome version should work again.
